tests/test_http_parser.py

```diff
@@ -117,6 +117,20 @@ def test_parse_headers(parser: Any) -> None:
     assert not msg.upgrade
 
 
+def test_parse_headers_longline(parser: Any) -> None:
+    text = (
+        b"""GET /test HTTP/1.1\r
+Test\xd9Header"""
```

Comment:
Could you put `b'\xd9'` into a dedicated variable (`invalid_unicode_byte`?) and interpolate it into this string? This way, it'd be cleaner what the intent is.